deploy/helm/sumologic/values.yaml

```diff
@@ -538,23 +538,23 @@ fluentd:
       ## Matching unit will be excluded from Sumo. The logs will still be sent to FluentD.
       excludeUnitRegex: ""
 
-    ## Extra Environment Values - allows yaml definitions
-    # extraEnvVars:
-    #   - name: VALUE_FROM_SECRET
-    #     valueFrom:
-    #       secretKeyRef:
-    #         name: secret_name
-    #         key: secret_key
-
-    # extraVolumes:
-    #   - name: es-certs
-    #     secret:
-    #       defaultMode: 420
-    #       secretName: es-certs
-    # extraVolumeMounts:
-    #   - name: es-certs
-    #     mountPath: /certs
-    #     readOnly: true
+      ## Extra Environment Values - allows yaml definitions
+      # extraEnvVars:
+      #   - name: VALUE_FROM_SECRET
+      #     valueFrom:
+      #       secretKeyRef:
+      #         name: secret_name
+      #         key: secret_key
+
+      # extraVolumes:
+      #   - name: es-certs
+      #     secret:
+      #       defaultMode: 420
+      #       secretName: es-certs
+      # extraVolumeMounts:
+      #   - name: es-certs
+      #     mountPath: /certs
+      #     readOnly: true
```

Comment:
this is incorrect indentation

Reply:
Sure, my bad.

Just wanted to note that when `comments-indentation` is enabled, `yamllint` complains about those lines (among others)

```
yamllint -c .yamllint.yaml deploy/helm/sumologic/values.yaml
deploy/helm/sumologic/values.yaml
  541:5     error    comment not indented like content  (comments-indentation)
  742:3     error    comment not indented like content  (comments-indentation)
  744:5     error    comment not indented like content  (comments-indentation)
  762:5     error    comment not indented like content  (comments-indentation)
  957:5     error    comment not indented like content  (comments-indentation)
  980:7     error    comment not indented like content  (comments-indentation)
  997:7     error    comment not indented like content  (comments-indentation)
  1010:7    error    comment not indented like content  (comments-indentation)
  1655:5    error    comment not indented like content  (comments-indentation)
  1795:7    error    comment not indented like content  (comments-indentation)
```

Reply:
Maybe some newline before comment would help

Reply:
🤷🏻 
for now I've left it disabled so that we can deal with this later

Reply:
sure, please create issue for that if it isn't not created already

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1178